charts/ditto/Chart.yaml

```diff
@@ -14,10 +14,9 @@ name: ditto
 description: |
   Eclipse Ditto™ is a technology in the IoT implementing a software pattern called “digital twins”.
   A digital twin is a virtual, cloud based, representation of his real world counterpart
-  (real world “Things”, e.g. devices like sensors, smart heating, connected cars, smart grids,
-  EV charging stations etc).
-version: 1.5.3
-appVersion: 1.5.1
+  (real world “Things”, e.g. devices like sensors, smart heating, connected cars, smart grids, EV charging stations etc).
+version: 1.5.4
```

Comment:
I would rather have the `version` be in sync with the `appVersion`, so please adjust to `2.0.0` here as well.

Reply:
that is a good idea to keep this in sync 👍🏻